src/main/java/com/sourcery/gymapp/backend/workout/model/Exercise.java

```diff
@@ -0,0 +1,45 @@
+package com.sourcery.gymapp.backend.workout.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Column;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+
+import java.util.List;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "exercise", schema = "workout_data")
+public class Exercise extends BaseEntity {
```

Comment:
probably I would add size validation to string fields here just in case we want to have possibility open to add new Exercises